### PR TITLE
rewrite code to common java way

### PR DESCRIPTION
**Pull request by ondrejvelisek**
class attributes ALWAYS private. Than you can manage access by getter and setter methods.
to print object we overwrite toString function. Than you can use only println(object) to print it.
for documentation classes and method is used /*\* instead of /\* before declaration. Look to code. Than there are documentation generators who can generate whole documentation for project from this "special" coments.
add .gitignore file (not sure if it will work, try to pull it)

In IntelliJ IDEA try to tap alt+insert in class editor. Idea will help you to generate some common code. ;-)

### CODE REVIEW

File: src/prop13/graphlibrary/Edge.java [hunk 1]
```diff
@@ -5,13 +5,33 @@
...
+
+	public Node getN2() {
+		return n2;
+	}
+
+	public int getWeight() {
+		return weight;
+	}
+
+	public void setN1(Node n1) {
```
Comment (miqueljubert) [inline src/prop13/graphlibrary/Edge.java]: I am not sure it's a good idea to let people change one of the nodes of an edge, it is not a needed functionality of the class and it is really problematic for the graph class (you could change one of the nodes of an edge without making any change to the graph). I think that the nodes of an edge shoud be setted with the constructor and impossible to change.

File: src/prop13/graphlibrary/Edge.java [hunk 2]
```diff
@@ -5,13 +5,33 @@
...
-    protected Node n2;
-    protected int weight;
+    private Node n1;
+    private Node n2;
+    private int weight;
 
-    public Edge()
-    {
-    }
+	public Node getN1() {
```
Comment (miqueljubert) [inline src/prop13/graphlibrary/Edge.java]: I didn't implement getters for n1 and n2 (and used that "weird" getNode and getNeighbor) to try to make both nodes have the same status on the edge (no origin or destiny, you can only get one the nodes of the edge in an arbitrary way, the other you have to get with getNeighbor) due to the fact that we will be working with undirected edges.

Comment (ondrejvelisek) [inline src/prop13/graphlibrary/Edge.java]: Yea, I get it. It's still only beginning. But I don't know what other teams need. We can implement undirected in higher layer (relatin, member, congress) But OK I will delete it.

File: src/prop13/graphlibrary/Edge.java (hunk 1 above)
Comment (ondrejvelisek) [inline src/prop13/graphlibrary/Edge.java]: Absolutely agree.

**Pull request merged**